src/utilities/MessageUtilities.ts

```diff
@@ -136,8 +137,8 @@ export namespace MessageUtilities {
     export function getMessageOptionsFromMessage(
         msg: Message,
         components?: MessageActionRow[]
-    ): MessageOptions & { split?: false | undefined } {
-        const obj: MessageOptions & { split?: false | undefined } = {
+    ): MessageEditOptions & { split?: false | undefined } {
```

Comment:
This function is ideally supposed to return a `MessageOptions` (think of it as it "extracting" the properties of a message back into a `MessageOptions` object, which then can be used to send another message).

It might be better to create a helper function that converts between `MessageOptions` and `MessageEditOptions`, instead, as it looks like `MessageEditOptions` is intended for `Message#edit` whereas `MessageOptions` is intended for `Message#send`. 

Let me know what you think.

Reply:
Fair. If needed we can always introduce a helper converter function.